### PR TITLE
Looked through the strings in the English translation to compare it t…

### LINKED ISSUE #254: Localization management system for node.js web application
We need a localization management system for this web application - running node.js.

Requirements:
- Open Source

- Ability to show what needs localization and into what languages - either automatically or by running a script

Nice to have:
- Warnings for hard-coded strings that should be internationalized.

- Auto-create issues for localization needs

### PR DESCRIPTION
**Pull request by rh9891**
…o the French translation making sure that the groupings were preserved. Inserted missing values into the translation file. Corrected grammatical errors. Those without a translation in the French were given the same value.

## Related Issue

Fixes #254 

## Checklist:

- [X] I have read the [contributor's guide](https://wildaid.github.io/contribute/index.html).
- [X] I linked an issue in the previous section
- [X] I have commented on the linked issue
- [X] I was assigned the linked issue (not required)
- [X] I have tested the change to the best of my ability against the [sandbox](https://wildaid.github.io/contribute/sandbox.html) or a [local build](https://wildaid.github.io/build).

Optional items:

- [ ] My change adds new text and requires a change to translations.
- [ ] My change requires a change to the documentation.
- [ ] I have submitted a PR to the [documentation repo](https://github.com/WildAid/wildaid.github.io).
- [ ] I was not able to test... (explain below, e.g. you did not have permissions to test a specific feature)
- [ ] This change depends O-FISH Realm repository changes (explain below)

* **Optional: Add any explanations here** 

* **Optional: Add any relevant screenshots here**

### CODE REVIEW

**Review by Sheeri — CHANGES_REQUESTED** [review]
Hi @rh9891 - I love the changes you've made - I'm going to go through them in a bit more detail than I have already - making sure the spelling fixes you've done don't break other things (e.g. make sure changing "ELECTRINIC_SYSTEM" to "ELECTRONIC_SYSTEM" doesn't need to be done in the code also, since it's in the translations).

However, the primary ask was for a script, that would compare the localizations and add anything missing to the bottom of the file. I described it here: https://github.com/WildAid/o-fish-web/issues/254#issuecomment-707861383

I love that you also added the changes, and we will definitely merge them once I do some more testing....but for the issue itself, we need the script (so that we can run it periodically).

Thanks!

File: src/helpers/i18n/en/translation.json [hunk 1]
```diff
@@ -164,7 +161,7 @@
     "SEE_ALL_RESULTS": "See all results for '{{searchQuery}}'"
   },
   "LOADING": {
-    "LOGGIN_IN": "Logging in...",
+    "LOGGING_IN": "Logging in...",
```
Comment (Sheeri) [inline src/helpers/i18n/en/translation.json:164]: Please also change in the referring file:
components/login/login.component.js

And in the Spanish translation:
src/helpers/i18n/es/translation.json

File: src/helpers/i18n/fr/translation.json [hunk 2]
```diff
@@ -86,50 +86,49 @@
...
-    "WHO_CAN_ACCESS_DATA": "",
-    "SHARING_DATA_WITH": "",
-    "FOLLOWING_AGENCIES": "",
-    "SHARING_DATA_WITH_ZERO_AGENCIES": "",
-    "AGENCY_ADMINS": "",
+    "SHARED_DATA": "Données Partagées",
+    "BOARDINGS_FROM": "BOARDINGS FROM",
+    "BOARDINGS_TO": "À",
+    "WHO_CAN_ACCESS_DATA": "Qui peut accéder aux données",
+    "SHARING_DATA_WITH": "{{agency}} informe {{sharingAgency}}",
```
Comment (Sheeri) [inline src/helpers/i18n/fr/translation.json:93]: This should be the translation of "Agencies sharing data with {{agency}}". (as above)

**Review by Sheeri — CHANGES_REQUESTED** [review]
Hi @rh9891,

I can see you put a LOT of work into this! It may seem like there are a lot of comments here, but that's because changing text can be fraught. 

For each change, I searched the code base for the string (e.g. 'LONGTITUDE') to see if the previous 'bad' variable was still used. In some cases, the 'bad' versions were, so I've noted what code should be updated.

I think the best way to proceed here would be to first fix the issues in this review of the PR. Then I can merge the PR. I will keep the referring issue open, and you can submit the script itself in a NEW pull request. (You absolutely deserve credit for 2 PR's given the amount of work you're putting in!)

Keep up the good work!

File: src/helpers/i18n/en/translation.json [hunk 3]
```diff
@@ -90,14 +90,12 @@
     "BOARDINGS_FROM": "Boardings From",
     "BOARDINGS_TO": "To",
     "WHO_CAN_ACCESS_DATA": "Who can access this data",
-    "SHARING_DATA_WITH": "Agencies sharing data with {{agency}}",
-    "FOLLOWING_AGENCIES": "{{agency}} is sharing boarding data with the following agencies",
+    "SHARING_DATA_WITH": "{{agency}} sharing data with {{sharingAgency}}",
```
[the remark this answers is not in the document]
Comment (rh9891) [reply src/helpers/i18n/en/translation.json]: I will be leaving this AS IS since the line was only changed up/down.

**Review by Sheeri — APPROVED** [review]

**Comment by Sheeri** [comment]
Phew! Looks great, @rh9891 ! 

I will merge this but keep the original issue open - please open a new branch for a script that will do the comparison of strings/translations. thanks!

**Pull request merged**